.github/workflows/release.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     needs:
       - lib-check
       - ci-tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04 # must match base in charmcraft.yaml
```

Comment:
since `latest == 22.04` what's the benefit of pinning here?
github will stick to lts only, and in worst cases, when we reach 24.04, not pinning it will be instrumental in warning about the necessity of an update.

Reply:
my $0.05:

> Although if we upgrade to 24.04 before github actions we'd have to pin the version at that point

Or prepare compatibility upfront.
In general I agree with you, however mass-pinning versions without a real need have list of opposite drawbacks.
At the moment we are using 'ubuntu-latest' image everywhere, I would keep it "identical" even if not yet "perfect".